Doc/library/asyncio-eventloop.rst

```diff
@@ -1519,7 +1525,7 @@ Wait until a file descriptor received some data using the
 Set signal handlers for SIGINT and SIGTERM
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-(This example only works on UNIX.)
+(This ``signals`` example only works on UNIX.)
```

Comment:
Not sure that this change adds to the clarity here, but up to you.

Reply:
Going to leave `signals` here since it clarifies for less experienced users why Windows will not work.